components/button.css

```diff
@@ -3,4 +3,6 @@
   background-color: #29282a;
   border: 0.1rem solid #e6664e;
   padding: 1rem;
+  width: 25%;
 }
+/*not sure how to position the button in the center of the card, any ideas?*/
```

Comment:
put a <div> or <span> element around the button.

Reply:
solved it with `align items: center;` 🥳